site/src/component/QuarterTooltip/Chart.tsx

```diff
@@ -153,7 +156,7 @@ export default class Chart extends React.Component<ChartProps> {
                 legendOffset: 36,
               }}
               axisLeft={{
-                tickValues: Array.from({ length: greatestCount }, (_, i) => i + 1), // integers from 1 to max
+                tickValues: Array.from({ length: greatestCount / tickSize }, (_, i) => i * tickSize + tickSize), // integers from 1 to max
```

Comment:
Change or remove the out-of-date comment
```suggestion
                tickValues: Array.from({ length: greatestCount / tickSize }, (_, i) => i * tickSize + tickSize),
```